test/parallel/test-timers-unenroll-unref-interval.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const common = require('../common');
+const timers = require('timers');
+
+const keepOpen = setTimeout(() => {
+  common.fail('Test timed out.');
+}, common.platformTimeout(500)); // Keep event loop open.
```

Comment:
I would much prefer this just keep the event loop open indefinitely and not throw:

```javascript
const keepOpen = setInterval(() => {}, 9999);
```

(Would need to change `clearTimeout()` below to `clearInterval()`.)

@nodejs/testing